src/map/clif.h

```diff
@@ -369,8 +369,6 @@ enum clif_messages {
 	MSG_MERCENARY_ESCAPED          = 0x4f5, ///< The mercenary has run away.
 	MSG_PARTY_MEMBER_NOT_SUMMONED  = 0x4c5, ///< The party member was not summoned because you are not the party leader.
 	MSG_PARTY_NO_MEMBER_IN_MAP     = 0x4c6, ///< There is no party member to summon in the current map.
-	MSG_SKILL_CANT_USE_AREA        = 0x536, ///< This skill cannot be used within this area
```

Comment:
remove known messages from msgstringtable probably not good idea.

Reply:
This message is useless now, since I already know the way how to show this properly.

Reply:
better left it for future usage.
more constants added here is better. if even unused now.
for old clients you can show this message from msgstringtable